Chamasma/DesignSystem/Sources/DesignSystem/Components/ComponentViews/MealCell.swift

```diff
@@ -92,12 +76,20 @@ public struct MealCell: View {
 struct MealCell_Previews: PreviewProvider {
     static var previews: some View {
         MealCell(
-            name: "ROYAL DOUBLE",
-            description: "King chicken, big king, onion tings, big fries, mini auce, coca-cola 1L.",
-            price: 123.45,
-            rating: 0.78,
-            image: DesignSystem.Image.burger,
-            isPopular: true
+            meal: .init(
+                name: "ROYAL DOUBLE",
+                description: "King chicken, big king, onion tings, big fries, mini auce, coca-cola 1L.",
+                rating: 0.78,
+                price: 123.45,
+                image: DesignSystem.Image.burger,
+                ingredients: [],
+                isVegetarian: false,
+                isVegan: false,
+                isGlutenFree: false,
+                isPopular: true,
+                isSpicy: true,
+                allergens: []
+            )
```

Comment:
I can not use Meal.example here because it is in Presentation module, As it (MealCell) is in DesignSystem and Presentation uses DesignSystem, it is circular dependency, can you suggest anything about this?